main.go

```diff
@@ -333,39 +376,36 @@ func createAuthAcquirer(v *viper.Viper) (acquire.Acquirer, error) {
 	return nil, errors.New("auth acquirer not configured properly")
 }
 
-func newTimeoutConfigs(v *viper.Viper) (t *timeoutConfigs, err error) {
-	var c, r, d time.Duration
-	if c, err = time.ParseDuration(v.GetString(clientTimeoutKey)); err == nil {
-		if r, err = time.ParseDuration(v.GetString(reqTimeoutKey)); err == nil {
-			if d, err = time.ParseDuration(v.GetString(netDialerTimeoutKey)); err == nil {
-				t = &timeoutConfigs{
-					cTimeout: c,
-					rTimeout: r,
-					dTimeout: d,
-				}
-			}
-		}
+func newTimeoutConfigs(v *viper.Viper) (timeoutConfigs, error) {
+	c, err := time.ParseDuration(v.GetString(clientTimeoutKey))
+	if err != nil {
+		return timeoutConfigs{}, err
 	}
-	return
-}
-
-func newClient(v *viper.Viper, t *timeoutConfigs) *http.Client {
-	return &http.Client{
-		Timeout: t.cTimeout,
-		Transport: &http.Transport{
-			Dial: (&net.Dialer{
-				Timeout: t.dTimeout,
-			}).Dial},
+	r, err := time.ParseDuration(v.GetString(reqTimeoutKey))
```

Comment:
It would be better if properties like this were part of a struct.  Why not just use a `TimeoutConfig` struct that's part of the candlelight config?